setup.py

```diff
@@ -26,7 +26,7 @@
         for l in Path('requirements.txt').read_text('utf-8').splitlines()
     ],
     python_requires='>=3.6',
-    packages=find_packages(),
+    packages=find_namespace_packages(include=['anndata.*']),
```

Comment:
I already did. It just finds all directories in there, which e.g. means that the tests of an installed anndata can actually be run because the directory with the necessary test data gets installed.

Reply:
While it seems to work fine for me, `asv` just totally fails to install this now. I'm pretty unclear on why.

Reply:
But it does work if I revert this to `packages=find_packages(),`, and add `anndata/core/__init__.py`

Reply:
I got asv to run in the meantime using the virtualenv backend. It works fine for me. What’s your error?

Reply:
I'm using the conda backend. It says it installs fine, but just fails to import anndata:

```python
Traceback (most recent call last):
File "/usr/local/lib/python3.7/site-packages/asv/benchmark.py", line 1315, in <module>
main()
File "/usr/local/lib/python3.7/site-packages/asv/benchmark.py", line 1308, in main
commands[mode](args)
File "/usr/local/lib/python3.7/site-packages/asv/benchmark.py", line 1044, in main_run
benchmark_dir, benchmark_id, extra_params=extra_params)
File "/usr/local/lib/python3.7/site-packages/asv/benchmark.py", line 939, in get_benchmark_from_name
module = import_module(modname)

File "/Users/isaac/github/anndata-benchmarks/.asv/env/8cb4204c4acf92f37ebde6042bb3eb94/lib/python3.7/importlib/__init__.py", line 127, in import_module
return _bootstrap._gcd_import(name[level:], package, level)
File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
File "<frozen importlib._bootstrap>", line 983, in _find_and_load
File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
File "<frozen importlib._bootstrap>", line 677, in _load_unlocked

File "<frozen importlib._bootstrap_external>", line 728, in exec_module
File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
File "/Users/isaac/github/anndata-benchmarks/benchmarks/readwrite.py", line 32, in <module>
    from .utils import (
File "/Users/isaac/github/anndata-benchmarks/benchmarks/utils.py", line 10, in <module>
    from anndata import AnnData
ImportError: cannot import name 'AnnData' from 'anndata' (unknown location)
```

Reply:
Pretty sure that means you have another version of anndata somewhere else on your PYTHONPATH. If you fix that it should work.

Reply:
There isn't. However, anndata (at a011d7efaa6) that get's installed by asv only has these files:

<details>
<summary> anndata installed by asv files </summary>


```bash
# The version installed by asv
$ cd ~/github/anndata-benchmarks/.asv/env/8cb4204c4acf92f37ebde6042bb3eb94/lib/python3.7/site-packages/anndata
# Listing the none `__pycache__` files
$ find . | grep -v "__pycache__" 
.
./readwrite
./readwrite/write.py
./readwrite/__init__.py
./readwrite/utils.py
./readwrite/zarr.py
./readwrite/h5ad.py
./readwrite/read.py
./_core
./_core/index.py
./_core/alignedmapping.py
./_core/raw.py
./_core/anndata.py
./_core/sparse_dataset.py
./_core/file_backing.py
./_core/views.py
./tests
./tests/test_utils.py
./tests/conftest.py
./tests/test_obsmvarm.py
./tests/test_inplace_subset.py
./tests/test_readwrite.py
./tests/test_layers.py
./tests/test_deprecations.py
./tests/test_io_conversion.py
./tests/test_views.py
./tests/test_raw.py
./tests/test_hdf5_backing.py
./tests/test_repr.py
./tests/test_helpers.py
./tests/test_base.py
./tests/helpers.py
./tests/test_uns.py
./tests/test_backed_sparse.py
./tests/test_obspvarp.py
```

</details>

Reply:
Ah, I see. Fixed!